workers/notifier/src/main/kotlin/notifier/NotifierOrtResultGenerator.kt

```diff
@@ -64,6 +64,9 @@ internal class NotifierOrtResultGenerator(
 
         /** The separator for multiple email recipients. */
         private const val RECIPIENTS_SEPARATOR = ";"
+
+        /** The label for the runId. */
+        private const val RUN_ID_LABEL: String = "runId"
```

Comment:
The class comment contains a description of the labels available to the notifier script. Please add documentation for this new label there as well.